app/models/article.rb

```diff
@@ -1,2 +1,4 @@
 class Article < ActiveRecord::Base
+  has_many  :comments
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.